### PR TITLE
Add a paragraph about running tests in single node environments.

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

File: cnf-tests/README.md [hunk 1]
```diff
@@ -446,6 +447,16 @@ To make it quicker, a `CLEAN_PERFORMANCE_PROFILE="false"` can be set to instruct
 docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e CLEAN_PERFORMANCE_PROFILE="false" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ```
 
+## Running in single node cluster
+
+Running tests on a single node cluster causes some limitations to be imposed:
+- longer timeouts for certain tests
+- some tests requiring multiple nodes are skipped
+
+The longer timeouts concern mostly SRIOV and SCTP tests. Reconfiguration requiring node reboots cause a reboot of the whole environment and hence takes longer to complete.
```
Comment (SchSeba) [inline cnf-tests/README.md:456]: whole environment including the control openshift control plane

Comment (mmirecki) [reply cnf-tests/README.md:456]: Done

Comment (SchSeba) [reply cnf-tests/README.md:456]: I don't see a change in the PR

**Review by SchSeba — APPROVED** [review]
/lgtm
/approve

**Pull request merged**